src/screens/CurrencySelectModal.js

```diff
@@ -158,20 +160,39 @@ export default function CurrencySelectModal() {
   }, [assetsToFavoriteQueue, handleApplyFavoritesQueue, searchQueryExists]);
 
   const handleFavoriteAsset = useCallback(
-    (assetAddress, isFavorited) =>
+    (asset, isFavorited) => {
```

Comment:
assetAddress is changed to asset, which means we should pass in the asset in ExchangeCoinRow's onFavoriteAsset

Reply:
@jinchung ya this change shouldnt be in this PR, my bad it belongs in my favoriting related PR

Reply:
@jinchung nvm it does need to be here too, i fixed it here https://github.com/rainbow-me/rainbow/pull/929/commits/97b9080e34acbcedc6bef2007cee4955bd5d87f0